packages/build/webpack/watch.config.js

```diff
@@ -17,18 +18,20 @@ __dirname = lkModule;
 const devServer = {
     host: 'localhost',
     port: constants.watchPort,
-
     // enable the HMR on the server
     hot: true,
-
     headers: {
         "Access-Control-Allow-Origin": "*",
         "Access-Control-Allow-Methods": "GET, POST, PUT, DELETE, PATCH, OPTIONS",
         "Access-Control-Allow-Headers": "X-Requested-With, content-type, Authorization"
     },
-
     compress: true,
-    overlay: true
+    overlay: true,
+    watchOptions: {
+        // Ignore any packages folders, if we don't ignore packages then we will incorrectly trigger builds in
+        // package folders (e.g. changing a file in the SM Workflow package would incorrectly trigger a build in SM)
+        ignored: ['**/packages']
```

Comment:
in my testing this morning with this fix_hmr changes plus the webpack alias from src changes, we will want to conditionalize this so that we don't ignore the package dirs when we are doing `npm run start-link`. this file should have access to that via `process.env.LINK`

Reply:
Do we want to do that in this PR or yours?

Reply:
Since this will impact the current start-link behavior, I believe, let's put it into this PR if it isn't too much trouble. I believe the change would look like this:
`ignored: process.env.LINK ? [] : ['**/packages']`

Reply:
Done.